news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,124 @@
+
+Each month, a selection of volunteers work alongside the Quality Assurance Team to pick out the best of the best for each mode. We reward the top beatmap in each category by providing **1 month of supporter status** to the mapper. **The May 2017 spotlight rankings will conclude on the 15th July 2017.**
+
+## Past Spotlights results
+
+First of all, congratulations to the winners of the April 2017 Spotlights. They receive one month of supporter status for their efforts!
+
++ **osu!:** [Toy](https://osu.ppy.sh/u/2757689)
++ **osu!taiko:** [Katdon_donKat](https://osu.ppy.sh/u/8089664)
++ **osu!catch:** [Crystal](https://osu.ppy.sh/u/1646397)
++ **osu!mania:** [Hestia-](https://osu.ppy.sh/u/393151)
+
+
+## osu!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/orvid.jpg)](https://osu.ppy.sh/s/578332)
+
+RLC easily took the top spot at this month's spotlight with this beautiful marathon. Reminding us of RLC's previous masterpieces such as Anhedonia, the music offers a lot of variety and interesting rhythms over the course of more than 10 minutes. Every track of the gapless album is mapped differently, but the map still feels coherent and polished throughout. Totally worth checking out.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/get-on.jpg)](https://osu.ppy.sh/s/554084)
+
+sionKotori continues to impress with his newly ranked map, full of very interesting patterns which fit very well with the song and look amazing, other than giving a pretty cool sense of speed. Recommended for everyone who enjoys technical and slider-heavy maps!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/glass-no-kutsu.jpg)](https://osu.ppy.sh/s/585681)
+
+Taking a spot in almost every spotlight lately, Delis has made it again. This time, combining the exceptional flow and stunning variety they are known for with an emotional song by Ito Kanako. The mapping perfectly represents the impactful changes of intensity provided by the song. This map is an experience you won't regret!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/bellow.jpg)](https://osu.ppy.sh/s/504315)
+
+I see sets like this more and more rarely as time passes, which is a shame. All the four mappers involved in the set did a splendid job in using the unusual doubles in this song and provided (although probably nothing exceptional) a really nice and clean approach to mapping. Overall this mapset is not pretentious at all, but that's most likely the reason why it is here, to demonstrate that not necessarily heavy rhythms or intense jump sessions are important to create an entertaining mapset that hooks the player. Whether you are an expert player or just a casual one you'll have something to chew on considering it is easily DT-able (not sure how many pp it gives tho :P). I could keep praising this mapset for a while but let's say that you should definitely give it a go and witness some old-school mapping modernized for the new standards.
```

Comment:
I guess I need to look up how official texts work... sorry.

What about...
``+ Doubles and low BPM songs are kind of rare nowadays, but fortunately we have [TicClick](https://osu.ppy.sh/users/672931)!
Providing us with a fairly simple yet effective interpretation, he manages to keep us going with the up-beat rhythm this music genre provides. Whether you are experienced or just started out playing the game you can bet there's something for you to enjoy, thanks to the low BPM allowing you to try out various mod combinations even at lower stages. This doesn't mean you should underestimate the set either though: some passages may turn out being trickier than expected. 
Definitely worth giving this mapset a try if you're looking for something unusual.``